metis_client/namespaces/datasources.py

```diff
@@ -19,20 +19,20 @@
 class MetisDatasourcesNamespace(BaseNamespace):
     """Datasources endpoints namespace"""
 
-    async def create_event(self, content: str) -> MetisRequestIdDTO:
+    async def create_event(self, content: str, fmt: str = None, name: str = None) -> MetisRequestIdDTO:
         "Create data source"
         async with self._client.request(
             method="POST",
             url=self._base_url,
-            json={"content": content},
+            json={"content": content, "fmt": fmt, "name": name},
```

Comment:
So, server-side formatting?
Now only "content" field is used by server. Is it ok?

Reply:
explained below in the referring commits